applications/views.py

```diff
@@ -218,6 +221,22 @@ def post(self, request, *args, **kwargs):
             status=status.HTTP_200_OK,
         )
 
+    def validate_linkedin_url(self, linkedin_url):
+        """
+        Validate that a LinkedIn URL has the right format and length
+
+        Args:
+            linkedin_url (string): LinkedIn URL of a user
+        """
+        if len(linkedin_url) > 200:
+            raise ValidationError(
+                {"errors": "The URL should be of 200 characters at max"}
```

Comment:
```suggestion
                {"errors": "The URL should be less than 200 characters."}
```

Trying to keep the message as simple as possible.